eng/pipelines/aggregate-reports.yml

```diff
@@ -26,10 +26,27 @@ jobs:
     pool:
       vmImage: 'windows-2019'
 
+    timeoutInMinutes: 360
```

Comment:
Does it really take 6 hours?

Reply:
It is currently taking less than 2 hours. I set this for testing.